addons/l10n_ae/models/account_chart_template.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models
+
+
+class AccountChartTemplate(models.Model):
+    _inherit = 'account.chart.template'
+
+    def _prepare_all_journals(self, acc_template_ref, company, journals_dict=None):
+        """ If UAE chart, we add 2 new journals TA and IFRS"""
+        if company.country_id.code == "AE":
+            if not journals_dict:
+                journals_dict = []
+            journals_dict.extend(
+                [{"name": "Tax Adjustments", "company_id": company.id, "code": "TA", "type": "general", "sequence": 1,
+                  "favorite": True},
+                 {"name": "IFRS 16", "company_id": company.id, "code": "IFRS", "type": "general", "favorite": True,
+                  "sequence": 10}])
+        return super()._prepare_all_journals(acc_template_ref, company, journals_dict=journals_dict)
+
+    def _load_template(self, company, code_digits=None, account_ref=None, taxes_ref=None):
+        account_ref, taxes_ref = super(AccountChartTemplate, self)._load_template(company=company,
+                                                                                  code_digits=code_digits,
+                                                                                  account_ref=account_ref,
+                                                                                  taxes_ref=taxes_ref)
+        if company.country_id.code == "AE" and self.get_xml_id().get(1) == "l10n_ae.uae_chart_template_standard":
```

Comment:
self.env.ref(... ==

Reply:
Do you mean I should replace this `if self == self.env.ref('l10n_ae.uae_chart_template_standard')`?

Reply:
Something like that, yes

Reply:
Alright I removed the condition